### PR TITLE
Add new sensor for uptime in seconds

### PR DESCRIPTION
**Pull request by CFenner**
Use *duration* device class and icon and uom as in other Uptime sensors in HA.

Bildschirmfoto 2024-01-29 um 10 03 50

### CODE REVIEW

**Comment by CFenner** [comment]
@hjelev what do you think, are you fine with changing the uom, or should I create a new topic/sensor for that?

**Comment by hjelev** [comment]
@CFenner add a new one pls, the old one with days is more readable if you are interested in the days.
The new format shows h:m:s right ?

**Comment by CFenner** [comment]
@hjelev added a new topic / sensor.

Currently this would fails for everyone as `uptime_seconds` is not defined. It would be nice if a missing config entry would be counted as `false`.

File: src/rpi-cpu2mqtt.py [hunk 1]
```diff
@@ -272,6 +273,12 @@ def config_json(what_config):
         data["name"] = "Uptime"
         data["unit_of_measurement"] = "days"
         data["state_class"] = "measurement"
+    elif what_config == "uptime_seconds":
+        data["icon"] = "mdi:timer-outline"
+        data["name"] = "Uptime"
+        data["unit_of_measurement"] = "s"
+        data["device_class"] = "duration"
+        data["state_class"] = "measurement"
```
Comment (CFenner) [inline src/rpi-cpu2mqtt.py:281]:
```suggestion
        data["state_class"] = "total_increasing"
```

**Comment by hjelev** [comment]
The update will take care of it.

**Pull request merged**